src/Lykke.blue.Service.ReferralLinks.Core/Assets/IAsset.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Lykke.blue.Service.ReferralLinks.Core.Assets
+{
+    public enum Blockchain
+    {
+        None,
+        Bitcoin,
+        Ethereum
+    }
+
+    public class Asset
```

Comment:
File name is "IAsset.cs", not "Asset.cs"

Reply:
Moreover, it is using only at ModelConverter, which is unused class. Remove this file.

Reply:
Done.